data/newtabicons-content.js

```diff
@@ -82,6 +84,7 @@ function updateThumbnails() {
         thumb.style.backgroundImage = thumb.dataset.newPreview;
         thumb.removeEventListener('mouseover', mouseOverListener);
         thumb.removeEventListener('mouseout', mouseOutListener);
+        window.addEventListener('mousemove', mouseMoveListener);
```

Comment:
I think so.. for the record, this switch statement doesn't match up with the 'toggle' one above for some reason. 'case 2' seems to be the 'Always On' option and I think that's the only case that needs the mousemove listener.